src/commands/monitors/run.rs

```diff
@@ -45,6 +45,37 @@ pub fn make_command(command: Command) -> Command {
                 .num_args(1..)
                 .last(true),
         )
+        .arg(
+            Arg::new("schedule")
+                .short('s')
+                .long("schedule")
+                .help("Configure the cron monitor with the given schedule (crontab format)"),
+        )
+        .arg(
+            Arg::new("checkin_margin")
+                .long("check-in-margin")
+                .value_parser(clap::value_parser!(u64))
```

Comment:
You could use
```rust
.requires("schedule")
```
here to automatically make it an error to supply this argument without `"schedule"`.

Reply:
Oh cool, didn't know about that! Should be able to simplify my code with that change.

Reply:
Making this change does change the behavior somewhat, since the way I initially wrote it, the `checkin_margin` and `max_runtime` just get ignored when `schedule` is missing, and we display a warning. But, maybe erroring is better than displaying a warning and proceeding, since people might miss a warning message

Reply:
Yes, I would definitely say so. If these options already existed and we didn't want to create errors for use cases that previously worked, that would be one thing, but we're newly adding all of this anyway.